### DISCUSSION TITLE
CMAKE 3.20 failing to build on Windows because of GIT ?

### DISCUSSION THREAD

**Discussion by shock-pro** [ginkgo-project/ginkgo#823] [Q&A] [answered]
Hello Everyone, I would need your help.
Here is the error that I get on Windows:
![image](https://user-images.githubusercontent.com/77303301/124652638-f47da100-de61-11eb-811c-5db7ea7340d9.png)

It appears that CMAKE can not find my GIT installation, then I defined two environment variables as highlighted, and still I have the same error. Is this because of GIT or it is something else ?
Can I bypass this step by defining another env variable or something else?
Also I noticed that by default it points to a location for HIP, when in fact HIP is not available on Windows, it is not on my computer for sure.
Thanks in advance!

**Comment by shock-pro** [comment, marked as answer]
Update: I was able to fix this issue by first installing the "Git for Windows",
after that I got an error related to Git certificates, so I had to disable this with:  
$ git config --global http.sslverify "false"

And that fixed the Git issue.
Cheers,

**Reply by yhmtsai** [reply]
glad to hear you already solve it.
and thanks for providing the solution!
if you have the error related to Git certificates, could you also paste it?
It will be helpful to others facing the same problem.

**Comment by shock-pro** [comment]
Hi Yuhsiang!
Here is the error about the certificates:
[image: image.png]
Fixed with:
$ git config --global http.sslverify "false"
Regards,

**Reply by yhmtsai** [reply]
Git seems not to forward the image from mail properly
